Code/GraphMol/SmilesParse/SmartsWrite.cpp

```diff
@@ -410,7 +410,19 @@ std::string getBondSmartsSimple(const Bond *bond,
   } else if (descrip == "BondInRing") {
     res += "@";
   } else if (descrip == "SingleOrAromaticBond") {
-    // don't need to do anything here... :-)
+    auto dir = bond->getBondDir();
+    switch(dir) {
+      case Bond::ENDDOWNRIGHT: {
+        res += "\\";
+        break;
+      }
+      case Bond::ENDUPRIGHT: {
+        res += "/";
+        break;
+      }
+      default:
+        break;
+    }
```

Comment:
Below there is a:

```
else if (descrip == "BondDir" && equery)
```

that we might need to fix, but I can't find anything that actually sets that query outside the QueryOps.h functions.